modules/nw-route-specific-annotations.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-specific-annotations_{context}"]
+= Route-specific annotations
+
+Using environment variables, a router can set the default
+options for all the routes it exposes. An individual route can override some
+of these defaults by providing specific configurations in its annotations.
+
+For all the items outlined in this section, you can set annotations on the
+*route definition* for the route to alter its configuration
+
+.Route annotations
+[cols="3*", options="header"]
+|===
+|Variable | Description | Environment variable used as default
```

Comment:
Here is one column named `Environment variable used as default` but as I've said user cannot update them directly, all ENVs are controlled by operator and they are not supported to update for now.
Maybe we can keep them as it but add one note for that, or delete the column.

Reply:
Thank you! I added a note following the table.